app/templates/macros/pagination.html

```diff
@@ -0,0 +1,45 @@
+{%- from "govuk_frontend_jinja/components/pagination/macro.html" import govukPagination -%}
+
+{% macro Pagination(params) %}
+    {% if params.data %}
+        {% set pagination = params.data %}
```

Comment:
```suggestion
{% macro Pagination(pagination) %}
```

I don't think we need to call the arg `params` and have the require a `data` key - just call iit `pagination` directly and use that?

Reply:
let me try

Reply:
Fixed